app/src/main/java/com/fstyle/androidtrainning/screen/main/ViewPagerAdapter.java

```diff
@@ -1,10 +1,10 @@
-package com.fstyle.androidtrainning.screen;
+package com.fstyle.androidtrainning.screen.main;
 
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentPagerAdapter;
 
-import com.fstyle.androidtrainning.MainActivity;
+import com.fstyle.androidtrainning.screen.main.MainActivity;
```

Comment:
WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#ImportFromSamePackage